R/format_data.R

```diff
@@ -1814,9 +1908,7 @@ fmt_datetime <- function(data,
           time <- gsub("am$", "AM", time)
         }
 
-        datetime <-
-          paste(date, time) %>%
-          tidy_gsub("NA NA", "NA")
+        datetime <- gsub("NA NA", "NA", paste(date, time))
```

Comment:
Let's create a helper/decorator function that prevents our formatter function from ever seeing NA inputs